local-tests/tests/wrapped-keys/testSignTransactionWithSolanaEncryptedKey.ts

```diff
@@ -125,7 +126,7 @@ export const testSignTransactionWithSolanaEncryptedKey = async (
     // const confirmation = await solanaConnection.confirmTransaction(signedTx);
     // console.log(confirmation); // { context: { slot: 321490379 }, value: { err: null } }
 
-    const signatureBuffer = Buffer.from(signedTx, 'base64');
+    const signatureBuffer = Buffer.from(ethers.utils.base58.decode(signedTx));
```

Comment:
This test was failing if using the LIT action code merged since #652, but passed because we were using the CID to run the tests.  Now that we run the tests against the current, in-repo LIT actions, it needed to be updated to match the changes made in #652.

Reply:
Shouldn't we use IPFS Cid in the tests too? https://github.com/LIT-Protocol/js-sdk/pull/675#discussion_r1792648904

Reply:
Once all the actions have been reviewed and we've published/pinned them to IPFS, we can remove the injection of the LIT action code here.  Right now it's using injected code so that we can be sure this commit is sane and functional even though it's not in IPFS yet (see other comment)